tests/CMakeLists.txt

```
@@ -201,6 +203,10 @@ if(WITH_PYTHON)
   add_fdb_test(TEST_FILES rare/TransactionTagApiCorrectness.toml)
   add_fdb_test(TEST_FILES rare/TransactionTagSwizzledApiCorrectness.toml)
   add_fdb_test(TEST_FILES rare/WriteTagThrottling.toml)
+  add_fdb_test(TEST_FILES rare/AllSimUnitTests.toml)
+  add_fdb_test(TEST_FILES rare/Downgrade.toml)
```

```suggestion
  add_fdb_test(TEST_FILES rare/Downgrade.toml IGNORE)
```
To unblock you, I would suggest you ignore this for now. This test is broken, I don't know whether it is still needed.